plugins/inputs/logparser/README.md

```diff
@@ -108,7 +108,9 @@ You must capture at least one field per line.
   - ts-"CUSTOM"
 
 CUSTOM time layouts must be within quotes and be the representation of the
-"reference time", which is `Mon Jan 2 15:04:05 -0700 MST 2006`
+"reference time", which is `Mon Jan 2 15:04:05 -0700 MST 2006`.  Both a comma
+and a period are supported as a number separator.
+EX: timestamp:ts-"2006-01-02 15:04:05.000" or timestamp:ts-"2006-01-02 15:04:05,000"
```

Comment:
While you can have a comma in the parsed string, in the `ts-""` format string it will have to be dot right now unless you replace it in the format string too.